docs/api/latest.md

```diff
@@ -1,4 +1,4 @@
-# API Docs - v1.0.10
+# API Docs - v1.0.11-SNAPSHOT
```

Comment:
remove this change..

Reply:
Fixed in 54d6aad47a857be0acfae27b1b7b3e2a187ab9a2